msal/src/main/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplication.kt

```diff
@@ -262,7 +262,7 @@ class NativeAuthPublicClientApplication(
                     GetAccountResult.AccountFound(
                         resultValue = AccountState.createFromAccountResult(
                             account = account,
-                            correlationId = DiagnosticContext.INSTANCE.threadCorrelationId,
+                            correlationId = "",
```

Comment:
Could you check what MSAL does here?

Reply:
Also, what was your thinking behind changing this from the thread ID to an empty string?

Reply:
Because of we used MSAL's GetCurrentAccountCommand inside getCurrentAccountInternal without our method getThreadCorrelationId() where we can transform UNSET into UUID. Also, since it has not null or empty string here, UNSET will slip through the chance inside initializeDiagnosticContext 
```
 final String correlationId = StringUtil.isNullOrEmpty(requestCorrelationId) ?
                    UUID.randomUUID().toString() :
```

And there is external correlation id passed into the command, it would apply the default value UNSET.